dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/common/hibernate/SoftDeleteHibernateObjectStore.java

```diff
@@ -53,7 +52,6 @@ public SoftDeleteHibernateObjectStore(
 
   @Override
   public void delete(@Nonnull SoftDeletableObject object) {
-    publisher.publishEvent(new ObjectDeletionRequestedEvent(object));
```

Comment:
Are you sure no one else but us was listening to this event?

Reply:
Yes.
This was fired to check what to do when a tracker entity was deleted and now all the methods like `deleteTrackerEntity` and `deleteEnrollment` in deletion handlers are removed.